Models/AddLeaderboardRequest.cs

```diff
@@ -0,0 +1,14 @@
+namespace backend.Models
+{
+    public class AddLeaderboardRequest
+    {
+        public string track {get;  set;}
+        public string name {get;  set;}
+        public int time {get;  set;}
```

Comment:
O padrão do dotnet é que atributos publicos sejam PascalCase
public string Track { get; set; }
Tens que instalar algum formatador de código na tua IDE pra que os espaços também fiquem corretos

Reply:
https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code-pt